app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficTableV2/index.js

```diff
@@ -251,23 +256,39 @@ const CurrentTrafficTable = ({
     };
   }, [isStaticPreview]);
 
-  const activeFiltersCount = useMemo(
-    () =>
-      [...trafficTableFilters.method, ...trafficTableFilters.statusCode, ...trafficTableFilters.resourceType].length,
-    [trafficTableFilters.method, trafficTableFilters.resourceType, trafficTableFilters.statusCode]
-  );
+  const activeFiltersCount = useMemo(() => {
+    const countFilters = (filters) => filters.reduce((acc, curr) => acc + curr.length, 0);
+
+    if (persistLogsFilters) {
+      const { method, statusCode, resourceType } = trafficTableFilters;
+      return countFilters([method, statusCode, resourceType]);
+    } else {
+      return countFilters([statusCodesFilters, methodTypeFilters, resourceTypeFilters]);
+    }
+  }, [
```

Comment:
add ts-expect error since it also expects the complete `trafficTableFilters` to be added to dependency array